embedding/cpp/elm_flutter_view.cc

```diff
@@ -54,13 +56,32 @@ bool ElmFlutterView::RunEngine() {
 }
 
 void ElmFlutterView::Resize(int32_t width, int32_t height) {
-  if (view_ && (width_ != width || height_ != height)) {
-    width_ = width;
-    height_ = height;
-    FlutterDesktopViewResize(view_, width_, height_);
+  assert(IsRunning());
+
+  int32_t view_width, view_height;
+  evas_object_geometry_get(evas_object_, nullptr, nullptr, &view_width,
+                           &view_height);
+  if (view_width != width || view_height != height) {
+    FlutterDesktopViewResize(view_, width, height);
   }
 }
 
+int32_t ElmFlutterView::width() {
```

Comment:
This is not an accessor.
```suggestion
int32_t ElmFlutterView::GetWidth() {
```

Reply:
Thanks. Done